doc/user/content/cloud/quickstart.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Materialize Cloud Quickstart"
+description: "Set up a Materialize Cloud account, create deployments, and connect to data sources."
+menu:
+  main:
+    parent: "cloud"
+    weight: 2
+---
+
+{{< cloud-notice >}}
+
+
+To get you started, we'll walk you through the following:
+
+1. Signing up for Materialize Cloud
+2. Creating a Materialize Cloud deployment
+3. Connecting to the deployment from a terminal
+4. Connecting to a data source
+5. Running sample queries
+
+## Sign up for Materialize Cloud
+
+Sign up at [https://cloud.materialize.com](https://cloud.materialize.com#signup).
```

Comment:
Might be redundant, but wdyt of copy/pasting "If you're interested in trying out Materialize Cloud, let us know." here as well?  To make it more obvious that we're invite-only right now

Reply:
That's in the the shared {{< cloud-notice >}} text, so it is actually on every page.